data/json/hobbies.json

```
@@ -1,4 +1,59 @@
 [
+  {
+    "type": "profession",
+    "subtype": "hobby",
+    "id": "driving_license",
+    "name": "Driving License",
+    "description": "You got your driving license and know how to drive a car.",
+    "points": 0,
+    "skills": [ { "level": 1, "name": "driving" } ]
+  },
+  {
+    "type": "profession",
+    "subtype": "hobby",
+    "id": "simple_home_cooking",
+    "name": "Simple Home Cooking",
+    "description": "You've been living on your own long enough that you know how to cook basic dishes and feed yourself.",
+    "points": 0,
+    "skills": [ { "level": 1, "name": "cooking" } ],
+    "proficiencies": [ "prof_food_prep" ]
+  },
+  {
+    "type": "profession",
+    "subtype": "hobby",
+    "id": "computer_literate",
+    "name": "Computer Literate",
+    "description": "Either through school, your job, or from your family you've learned the basis of how to use a computer.",
+    "points": 0,
+    "skills": [ { "level": 1, "name": "computer" } ]
+  },
+  {
+    "type": "profession",
+    "subtype": "hobby",
+    "id": "socially_adequate",
+    "name": "Socially Adequate",
+    "description": "You've spent a significant portion of your life living amongs other people here in New England and are familiar with the local social norms. You know how to talk with other people, and how to convey your emotions and understand those of others.",
```

```suggestion
    "description": "You've spent a significant portion of your life living among other people and are familiar with social norms. You know how to talk with other people, and how to convey your emotions and understand those of others.",
```
Typo fix, plus reduce specificity about location so that background also works for characters not from NE